templates/invoice.html

```diff
@@ -0,0 +1,343 @@
+{% load price from taxed_prices %}
+<!DOCTYPE html>
+<html>
+
+<head>
+    <style>
+        @page {
+            background-color: #FAFAFA;
+            margin: 0.5cm;
+        }
+
+        .section-header,
+        .section-invoice-info {
+            background-color: #EFF5F8;
+            height: 235px;
+        }
+
+        .section-left {
+            width: 50%;
+            float: left;
+        }
+
+        .section-right {
+            width: 50%;
+            float: right;
+        }
+
+        .header-category {
+            font-family: Inter;
+            font-style: normal;
+            font-weight: normal;
+            font-size: 14px;
+            letter-spacing: 0.02em;
+            color: rgba(40, 35, 74, 0.6);
+
+        }
+
+        .header-item {
+            display: block;
+            color: black;
+            font-weight: bold;
+
+            font-family: Inter;
```

Comment:
Are you sure it loaded? I see a serif font. Also, it is always good practice to set a fallback font, sans-serif here

Reply:
It look like so but I will check again, weasyprint ignores a lot of stuff unfortunately :/

Reply:
Check the logs, it tells a lot

Reply:
It is loaded properly, also added fallback font as suggested (in body tag), logs don't always say something is ignored, weasyprint is not a full rendering engine therefore some stuff is simply not working as expected. But managed to make it look all right.